src/components/AccountAssets/AccountAssetsList.tsx

```diff
@@ -8,14 +8,14 @@ import { AccountSpecifier } from 'state/slices/accountSpecifiersSlice/accountSpe
 import { AssetAccountRow } from '../AssetAccounts/AssetAccountRow'
 /**
  * This returns the assets inside an account
- * @param assetIds An array of CAIP19s for the account
+ * @param assetIds An array of CAIP-19 asset identifiers for the account
```

Comment:
This is called `asset specifiers` everywhere else, is the different terminology on purpose?

Reply:
@gomesalexandre not quite - account specifiers (because they're not truly account IDs as per caip10), but assetIds

Reply:
https://github.com/shapeshift/web/pull/1579/files#diff-17a2bc13569ab7a5198bb0d3154b018a812a172e051eea9570ff604fb34d8ca0R44 so probably these should be `asset identifiers` as well

Reply:
Fixed!